bears/stylus/StylintBear.py

```diff
@@ -36,12 +42,250 @@ class StylintBear:
+            names. If ``strict_naming_convention`` is set to ``True``,
+            ``variable_naming_convention`` will also look at class and
+            ID names.
+        :param none_keyword:
+            If ``True`` check for places where ``none`` used instead of ``0``.
+            If ``False`` check for places where ``0`` could be used
+            instead of ``none``.
+        :param check_no_important_keyword:
+            If ``True``, show warning when ``!important`` is found.
+            For example: If set to ``True``, this will throw a warning::
+
+                div
+                    color red !important
+
+        :param spaces_inside_parentheses:
+            Enforce or disallow use of extra spaces inside parentheses.
+            For example: If set to ``True``, prefer
+            ``my-mixin( $myParam )`` over ``my-mixin($myParam)``
+            or if set to ``False``, prefer
+            ``my-mixin($myParam)`` over ``my-mixin( $myParam )``.
+        :param placeholder:
+            Enforce extending placeholder vars when using ``@extend(s)``.
+        :param prefix_vars_with_dollar:
+            Enforce use of ``$`` when defining a variable. In Stylus, using a
+            ``$`` when defining a variable is optional, but is a good idea
+            if you want to prevent ambiguity. Not including the ``$`` sets up
+            situations where you wonder: "Is this a variable or a value?"
+            For instance: ``padding $default`` is easier to understand than
+            ``padding default``.
+            For example: If set to ``True``, prefer
+            ``$my-var = 0`` over ``my-var = 0``
+            or if set to ``False``, prefer
+            ``my-var = 0`` over ``$my-var = 0``.
+        :param semicolons:
+            When ``True``, enforce semicolons. When ``False``, disallow
+            semicolons.
+        :param sort_order:
+            Enforce a particular sort order when declaring properties. Throws
+            a warning if you don't follow the order.
+            For example: If set to ``alphabetical``, prefer this::
+
+                .some-class
+                    display block
+                    float left
+                    position absolute
+                    right 10px
+                    top 0
+
+            over this::
+
+                .some-class
+                    position absolute
+                    top 0
+                    right 10px
+                    display block
+                    float left
+
+            Supported values are ``alphabetical`` and ``grouped``.
+        :param stacked_properties:
+            No one-liners. Enforce putting properties on new lines.
+            For example: If set to ``False``, prefer::
+
+                .className
+                    padding 0
+
+            over
+            ``.className { padding 0 }``
+        :param check_valid_property:
+            Check that a property is valid CSS or HTML.
```

Comment:
ah sorry for missing this: we should call this setting `check_property_validity`, as we don't check valid-properties for something, but we check that they are actually valid :)